include/LightGBM/cuda/cuda_utils.h

```diff
@@ -1,24 +1,188 @@
+/*!
+ * Copyright (c) 2021 Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE file in the project root for license information.
+ */
+
 /*!
  * Copyright (c) 2020 IBM Corporation. All rights reserved.
```

Comment:
I believe we can simplify this.
```suggestion
/*!
 * Copyright (c) 2020-2021 IBM Corporation, Microsoft Corporation. All rights reserved.
```

Reply:
Resolved via f44b881